sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -54,23 +54,42 @@ public virtual string AccountName
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="TableClient"/> using the specified <see cref="Uri" /> <see cref="AzureSasCredential"/>.
+        /// Initializes a new instance of the <see cref="TableClient"/> using the specified <see cref="Uri" /> which contains a SAS token.
+        /// See <see cref="GetSasBuilder(TableSasPermissions, DateTimeOffset)" /> for creating a SAS token.
+        /// </summary>
+        /// <param name="endpoint">
+        /// A <see cref="Uri"/> referencing the table service account.
+        /// This is likely to be similar to "https://{account_name}.table.core.windows.net/" or "https://{account_name}.table.cosmos.azure.com/".
```

Comment:
Would this have a different URL pattern due to the assumed embedding of the SAS and table name?

Reply:
Good question - I'll update the example to reflect the additional information.